packages/snaps-rpc-methods/src/restricted/dialog.ts

```diff
@@ -37,16 +38,30 @@ import { type MethodHooksObject } from '../utils';
 
 const methodName = 'snap_dialog';
 
+export const DIALOG_APPROVAL_TYPES = {
+  [DialogType.Alert]: `${methodName}:alert`,
+  [DialogType.Confirmation]: `${methodName}:confirmation`,
+  [DialogType.Prompt]: `${methodName}:prompt`,
+  default: `${methodName}:default`,
```

Comment:
```suggestion
  default: `${methodName}`,
```
Potentially just no subtype, WDYT?